### PR TITLE
Deploying Keystone to Heroku

### PR DESCRIPTION
**Pull request by ropaolle**
First of all I love what you are doing with KeystoneJS.

As a new user I had some issues figuring out how to deploy Keystone to a free service. And as a lot is going on that was expected. I tried to deploy a starter project and was stuck. Anyway, in the end I did some testing and documented how to deploy the starter kits to Heroku.

PS
I am Swedish and are not sure if the English is ok, but the step by step instructions may still be useful for others.

### CODE REVIEW

**Comment by Vultraz** [comment]
You'll need to change the meta header at the top too so it actually displays on the docs site and in the correct section:
```js
<!--[meta]
section: guides
title: Heroku
subSection: deployment
[meta]-->
```

(Shortened the title since we seem to be going for shorter titles recently)

File: docs/guides/heroku.md [hunk 1]
```diff
@@ -1,9 +1,132 @@
...
+The Heroku app start log can be reached by pressing [More] and View logs.
+
+    2020-03-25T07:19:04.841481+00:00 app[web.1]: User created:
+    2020-03-25T07:19:04.841483+00:00 app[web.1]:   email: admin@example.com
+    2020-03-25T07:19:04.841483+00:00 app[web.1]:   password: a04ecbcb963d
+    2020-03-25T07:19:04.841483+00:00 app[web.1]: Please change these details after initial login.
+
+#### Use secure cookies
+
+To use secore coocies we need to add below to index.js.
```
Comment (timleslie) [inline docs/guides/heroku.md]:
```suggestion
To use secure cookies we need to add below to index.js.
```

Comment (ropaolle) [reply docs/guides/heroku.md]: Woops. Ok fixed.

File: docs/guides/heroku.md [hunk 2]
```diff
@@ -1,9 +1,132 @@
...
+
+And that you have created your first KeystoneJS app:
+
+- If not, see the guide [Your first KeystoneJS app in 5 minutes](https://www.keystonejs.com/quick-start/).
+
+At the moment it is only possible to deploy the TODO starter project without any modifications. Deploying [the STARTER starter project](#deploy-the-starter-starter-project) or [the BLANK starter project](#deploy-the-blank-starter-project) requires a few tweaks.
+
+## MongoDB Atlas
+
+MongoDB Atlas has a free tire that we can use as our database service. You need to create an account, but you can use the free tire.
```
Comment (Vultraz) [inline docs/guides/heroku.md]: All these should be "trial" nor "tire".

Comment (MadeByMike) [reply docs/guides/heroku.md:26]: Pretty sure "free tier" is a valid term

File: docs/guides/heroku.md [hunk 3]
```diff
@@ -1,9 +1,132 @@
...
 
-In this guide we will learn how to deploy our Keystone system to Heroku.
+This quick start guide shows how to deploy your first KeystoneJS app to Heroku and use MongoDB Atlas as our database service.
+
+## Requirements
+
+Before we start, make sure you have:
+
+- [Git](https://git-scm.com/downloads) installed with `git --version`.
+- [Docker](https://docs.docker.com/install/) installed with `docker -v`.
```
Comment (MadeByMike) [inline docs/guides/heroku.md:16]: Docker is listed here as a requirement but never mentioned in the guide. I personally don't use docker to deploy to Heroku but rather connect it to a github repo or use the CLI.

Do we need to mention docker?

File: docs/guides/heroku.md [hunk 4]
```diff
@@ -1,9 +1,132 @@
...
+The Heroku app start log can be reached by pressing [More] and View logs.
+
+    2020-03-25T07:19:04.841481+00:00 app[web.1]: User created:
+    2020-03-25T07:19:04.841483+00:00 app[web.1]:   email: admin@example.com
+    2020-03-25T07:19:04.841483+00:00 app[web.1]:   password: a04ecbcb963d
+    2020-03-25T07:19:04.841483+00:00 app[web.1]: Please change these details after initial login.
+
+#### Use secure cookies
+
+To use secure cookies we need to add below to index.js.
```
Comment (MadeByMike) [inline docs/guides/heroku.md:117]: This defaults to true where `node_env==="production"` which I think it does in Heroku by default. Worth mentioning how to configure it though 👍

File: docs/guides/heroku.md (hunk 2 above)
Comment (Vultraz) [reply docs/guides/heroku.md:26]: Yes, but it says "tire" (as in wheel) not "tier".

File: docs/guides/heroku.md (hunk 3 above)
Comment (ropaolle) [reply docs/guides/heroku.md:16]: You are correct. I used both, but not in the guide. The Docker requirement is not needed.

File: docs/guides/heroku.md (hunk 2 above)
Comment (ropaolle) [reply docs/guides/heroku.md:26]: Same same for a Swed 😄. I think the Atlas service is called tier and Heruko trial. I'll check and update.

File: docs/guides/heroku.md (hunk 4 above)
Comment (ropaolle) [reply docs/guides/heroku.md:117]: Correct. However, not really related to this guide, but as it as now everybody end up with the same cookie secret "qwerty" if they forget to change it, so I am not sure defaulting secure cookies to true in production is the best solution.

File: .changeset/slimy-birds-film.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
+'@arch-ui/docs': minor
+---
+
+Deploying Keystone to Heroku
```
Comment (MadeByMike) [inline .changeset/slimy-birds-film.md:5]:
```suggestion
'@keystonejs/website': patch
---

Added a guide on deploying Keystone to Heroku.
```

File: docs/guides/heroku.md [hunk 6]
```diff
@@ -1,9 +1,131 @@
...
+
+Before we start, make sure you have:
+
+- [Git](https://git-scm.com/downloads) installed with `git --version`.
+
+And that you have created your first KeystoneJS app:
+
+- If not, see the guide [Your first KeystoneJS app in 5 minutes](https://www.keystonejs.com/quick-start/).
+
+At the moment it is only possible to deploy the TODO starter project without any modifications. Deploying [the STARTER starter project](#deploy-the-starter-starter-project) or [the BLANK starter project](#deploy-the-blank-starter-project) requires a few tweaks.
```
Comment (MadeByMike) [inline docs/guides/heroku.md:21]:
```suggestion
Before we start, make sure you have [Git](https://git-scm.com/downloads) installed with `git --version`, and that you have created your first KeystoneJS app.

If not, see the guide [Your first KeystoneJS app in 5 minutes](https://www.keystonejs.com/quick-start/).

At the moment it is only possible to deploy the TODO starter project without any modifications. Deploying [the STARTER starter project](#deploy-the-starter-starter-project) or [the BLANK starter project](#deploy-the-blank-starter-project) requires a few tweaks.
```

Comment (MadeByMike) [reply docs/guides/heroku.md:21]: I'd suggest making these sentences\paragraphs rather than single item lists.

**Pull request merged**